src/+groundwater/calculateGroundwaterRecharge.m

```diff
@@ -97,6 +98,10 @@ soilThick               cumulative soil layers thickness (from top to bottom)
         gwfluxes.recharge = 0;
     end
 
+    % Get rechrage temperature
+    TT_flip(1:1:ModelSettings.NN) = SoilVariables.TT(ModelSettings.NN:-1:1);
+    gwfluxes.rechargeTemp = TT_flip(indxRchrg);
+
```

Comment:
Instead, can we just expose `indxRchrg` to BMI, so that any other values can be also retrieved in the future without changing the code here? Note that `SoilVariables` is already available through BMI.

Reply:
> Instead, can we just expose `indxRchrg` to BMI, so that any other values can be also retrieved in the future without changing the code here? Note that `SoilVariables` is already available through BMI.

I change upon your suggestion already, Thanks